ArduPlane/servos.cpp

```diff
@@ -289,6 +289,46 @@ void Plane::dspoiler_update(void)
     SRV_Channels::set_output_scaled(SRV_Channel::k_dspoilerRight2, dspoiler_inner_right);
 }
 
+/*
+ set airbrakes based on reverse thrust and/or manual input RC channel
+ */
+void Plane::airbrake_update(void)
+{
+    // Calculate any manual airbrake input from RC channel option.
+    int8_t manual_airbrake_percent = 0;
+    
+    RC_Channel *airbrake_in_ch = rc().find_channel_for_option(RC_Channel::AUX_FUNC::AIRBRAKE);
```

Comment:
find_channel_for_option() is quite expensive, better to do it in set_control_channels() called from one_second_loop().
I notice the flapin channel has the same problem, maybe you could fix that too?

Reply:
>maybe you could fix that too

good idea for a new PR! Lets get it right for this PR first and then translate it over. As you noticed, this is the same structure as set_servos_flaps()

Reply:
@tridge Are you preferring this PR be blocked until that rework is done?

Reply:
yes, it's a tiny change. Just declare airbrake_in_ch in Plane class along with the other channels (roll, pitch etc) and update in set_control_channels()

Reply:
Could we use an RC channel option instead?

Reply:
@samuelctabor I talked to @tridge and it's just about caching the RC_Channel pointer at 1Hz along with all the other channels in set_control_channels()